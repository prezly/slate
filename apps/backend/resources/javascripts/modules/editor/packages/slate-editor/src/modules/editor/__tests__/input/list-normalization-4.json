{
    "type": "document",
    "children": [
        {
            "type": "list-item",
            "children": [
                {
                    "type": "list-item-text",
                    "children": [
                        {
                            "text": "Hello"
                        }
                    ]
                },
                {
                    "type": "list-item",
                    "children": [
                        {
                            "type": "list-item-text",
                            "children": [
                                {
                                    "text": ""
                                }
                            ]
                        }
                    ]
                }
            ]
        }
    ],
    "version": "0.50"
}
