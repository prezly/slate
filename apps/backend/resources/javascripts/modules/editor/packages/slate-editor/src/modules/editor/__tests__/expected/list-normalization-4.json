[
    {
        "type": "paragraph",
        "children": [
            {
                "text": "Hello"
            }
        ]
    },
    {
        "type": "paragraph",
        "children": [
            {
                "text": ""
            }
        ]
    }
]
